gitdir/gitdir.py

```diff
@@ -150,7 +154,7 @@ def main():
 
     parser.add_argument('--flatten', '-f', action="store_true",
                         help='Flatten directory structures. Do not create extra directory and download found files to'
-                             ' current directory.')
+                             ' output directory. (default to current directory if not specified)')
```

Comment:
updated --flatten help info. It wasn't updated with the addition of the optional output directory